docs/en/operation/Scalability-Tuning.md

```diff
@@ -45,6 +45,19 @@ backup duration.
 time proportional to the number of files in the subtree being operated on. This should not impact
 user experience unless the subtree is significantly large (> 10k files).
 
+Other metrics related to the number of files and the number of blocks
+* Master.TotalPath
+* Master.InodeHeapSize (Estimate only)
+* Master.TotalBlocks
```

Comment:
I think this one is replaced by Master.UniqueBlocks and Master.BlockReplicaCount?